v19.2/set-vars.md

```diff
@@ -64,11 +64,9 @@ default database for the current session:
 ~~~
 
 ~~~
+  database
 +----------+
-| database |
-+----------+
-| bank     |
-+----------+
```

Comment:
I'm glad you're updating these examples opportunistically, but it reminds me that gosh I really miss the old wrapped tables output.